pkg/api/bzz.go

```diff
@@ -96,12 +97,69 @@ func (s *server) bzzDownloadHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	me, err := m.Lookup(path)
+	if pathVar == "" {
+		logger.Tracef("bzz download: handle empty path %s", address)
+
+		indexDocumentSuffixKey := bzzDownloadHandlerManifestRedirect(m, manifestWebsiteIndexDocumentSuffixKey)
+		if indexDocumentSuffixKey != "" {
+			u := r.URL
+			u.Path += "/"
+			u.Path += indexDocumentSuffixKey
+			redirectURL := u.String()
+
+			logger.Debugf("bzz download: redirecting to %s: %v", redirectURL, err)
+
+			http.Redirect(w, r, redirectURL, http.StatusTemporaryRedirect)
```

Comment:
temporary?

Reply:
Yes, that is great question.
I started with temporary redirect, but question should it be replaced by permanent, or just serve the content of `index-document` content ?

Reply:
As this will redirect `/` to `/index.html`, I think that we should serve the index document, not redirect.